tools/Azure.SDK.ChangelogGen/Azure.SDK.ChangelogGen.Tests/Azure.SDK.ChangelogGen.Tests.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Could you update the name of this directory to just be "tools/changelog-gen/" so it matches the other tool structures. Note, Once this is updated we will also need to update the pipeline to point at the new ci.yml location.

Reply:
I'm good with a alternative name I would just prefer the directory name under tools not be the duplicate name of the package it contains. I'm ok with "net-changelog-gen" or something like that as well.

Reply:
sorry for the delay response. how about changelog-gen-for-net? same pattern as azure-sdk-for-net. thanks.

Reply:
@weshaggard, @jsquire , i have updated the folder name to changelog-gen-for-net. Could you help to update the pipeline if you are fine with the name? thanks.

Reply:
Let's move to `net-changelog-gen-mgmt` to conform to the naming patterns in the `tools` directory and focus the intent on the management libraries.

Reply:
renamed to net-changelog-gen-mgmt